test/multiverse/suites/unicorn/config/newrelic.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Should we include the license header for YAML files? Currently only .rb and .rake files are checked for it.

Reply:
Looking at New Relic's guidelines for Open Source repositories ([this one is the most recent - Internal](https://docs.google.com/document/d/1g4v0LAro5dqEsKqr3DiIuFIhhWvEJhuPu_jgM6lCFS0/edit#)), it seems the license header is needed only on source files. What qualifies as a source file is not defined. 

My hunch would be that source files are what we distribute in the `lib` directory to our customers. This would lead me to believe that we don't need license headers in any of our test suite files. 

I'd be happy to add them to the YAML files if we prefer to err on the side of over-commenting.

Reply:
Having recently looked at the automated license checker, our license header is fresh in mind.

Good thoughts, thanks. I personally would prefer no headers at all on any files, and simply use the existence of the `LICENSE` file in the root of the project. For any type of relaying of individual code (if we were to attach it to an email, put it in a gist, etc.), then we might want to put the header at the top in lieu of also relaying the entire `LICENSE` file, but as long as everything is in the repo I think the header is overkill. It's especially bothersome that our header simply says to open up `LICENSE` as opposed to letting people with a standalone file-with-header in front of them to at least know that it's Apache 2 that's involved.